UNAVCO2022/0.4_Geographic_Information_Science_using_GDAL/01_IntroToRasterData.ipynb

```diff
@@ -72,19 +72,22 @@
    "metadata": {},
```

Comment:
<p>Maybe remove this cell completely?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/parosen/Geo-SInC/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Good idea. I sent a new commit</p>